tools/ci/illegal_dme_files.py

```diff
@@ -0,0 +1,43 @@
+# Naive search for illegal files. Has no semantic knowledge, just a lexical
```

Comment:
define illegal here, it seems like files that should never be included in a DME?